### PR TITLE
Changes for compatibility

### PR DESCRIPTION
**Pull request by barlebino**

### CODE REVIEW

File: src/Engine/Component/CameraComponents/CameraComponent.cpp [hunk 1]
```diff
@@ -26,11 +26,15 @@ CameraComponent::CameraComponent(float fov, float near, float far) :
 void CameraComponent::init() {    
     float fovRadians;
     fovRadians = glm::radians(m_fov);
+    fovRadians *= 1.2f;
```
Comment (jaafersheriff) [inline src/Engine/Component/CameraComponents/CameraComponent.cpp]: Pls can we fix this

File: src/Engine/Component/CameraComponents/CameraComponent.cpp [hunk 2]
```diff
@@ -26,11 +26,15 @@ CameraComponent::CameraComponent(float fov, float near, float far) :
...
 
-    farPlaneHeight = 2 * glm::tan(fovRadians / 2) * m_far;
+    float planeScale;
+    planeScale = 1.2f;
```
Comment (jaafersheriff) [inline src/Engine/Component/CameraComponents/CameraComponent.cpp]: Pls can we fix this

Comment (Daskie) [reply src/Engine/Component/CameraComponents/CameraComponent.cpp]: It's fixed on my branch that I'll make a pr for sometime today

File: src/Engine/Util/Util.cpp [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#include "Util.hpp"
+
+constexpr double Util::PI;
```
Comment (Daskie) [inline src/Engine/Util/Util.cpp]: what is this for? this isn't how constexpr works

Comment (barlebino) [reply src/Engine/Util/Util.cpp]: Without this file, compiler gives me the following:

- undefined reference to 'Util::infinity'

The same error happened with the constexpr in CollisionSystem.cpp.

Solution to problem was found here: https://stackoverflow.com/questions/8016780/undefined-reference-to-static-constexpr-char

More detail:

src/Engine/Component/CollisionComponents/CollisionComponent.cpp.o: In function `(anonymous namespace)::detMeshSpan(int, glm::tvec3<float, (glm::precision)0> const*)':
CollisionComponent.cpp:(.text+0x1025): undefined reference to `Util::infinity'
collect2: error: ld returned 1 exit status

File: src/Engine/Component/CameraComponents/CameraComponent.cpp (hunk 1 above)
Comment (barlebino) [reply src/Engine/Component/CameraComponents/CameraComponent.cpp]: I gotchu fam

File: src/Engine/Util/Util.cpp (hunk 3 above)
Comment (Daskie) [reply src/Engine/Util/Util.cpp]: right, i forgot constexpr wasn't super fleshed out in c++11. What constexpr is supposed to mean is inline and compile-time. Oh well

File: src/Engine/Component/CameraComponents/CameraComponent.cpp (hunk 2 above)
Comment (barlebino) [reply src/Engine/Component/CameraComponents/CameraComponent.cpp]: Fixed on my branch as well. Feel free to select which implementation to keep

**Pull request merged**